data/src/main/java/com/antosik/benchproject/data/popular/movies/entity/MovieEntity.kt

```diff
@@ -0,0 +1,16 @@
+package com.antosik.benchproject.data.popular.movies.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "movie")
+internal data class MovieEntity(
+    @PrimaryKey(autoGenerate = false)
```

Comment:
Why `autoGenerate = false`? You will manage ids by yourself?

Reply:
I thought it would be the same as id for the concrete movie because it's unique. For example: 
![screenshot_database](https://user-images.githubusercontent.com/57491794/212933310-fb0030e2-3448-4b88-a8d7-092ce3dab123.png)

Reply:
If you think it's not a good way to manage IDs I could delete it

Reply:
Hmmm... maybe in this case you can use the id from API, it should be unique. `autoGenerate` is good in most cases, but in this case, when you will map from entity to other model you will have a different id than from the API, so I think that I need to change my mind in this case and it will be better to use your id, like you did initially 😄